### ISSUE TITLE
Review div subtype values not enumerated in frus.sch

### ISSUE THREAD

**Issue by joewiz** [HistoryAtState/frus#299]
The [frus.sch](https://github.com/HistoryAtState/frus/blob/master/schema/frus.sch) Schematron file enumerates the allowed values for `div/@subtype` values. The list there is drawn from the values that are in the XSL scripts that enrich divs with `@subtype` values. 

A query of volumes shows that we have made use of several additional values; here are the values and the number of instances of each in the corpus:

1. chapter-introduction (8)
2. errata_document-numbering-error (2)
3. event (38)
4. graphic-materials (10)
5. note-on-covert-actions (39)
6. section (2)
7. subsection (85)
8. undetermined (3)

The following XPath leads to the individual instances and can be run in oXygen's XPath toolbar:

```xquery
//div[@subtype and not(@subtype = ('about-frus-series', 'acknowledgements', 'additional-volumes', 'appendix', 'editorial-note', 'editorial-policies', 'errata', 'historian-statement', 'historical-document', 'index', 'maps', 'notes', 'preface', 'press-release', 'referral', 'sources', 'table-of-contents', 'volume-summary'))]
```

For each of these values we need to either (a) add the value to the list of allowed `@subtype` values or (b) update the value to be one of the allowed values.

**Comment by WaxCylinderRevival** [comment]
(Note to self: related issue https://github.com/HistoryAtState/frus/issues/199)

**Comment by WaxCylinderRevival** [comment]
As of January 7, 2021:

Current @subtype value | frequency
-- | --
about-frus-series | 39
acknowledgements | 2
additional-volumes | 39
appendix | 19
editorial-note | 8,491
editorial-policies | 4
errata | 34
graphic-materials | 11
historian-statement | 0
historical-document | [_forthcoming_]
index | [_forthcoming_]
notes | 4
note-on-covert-actions | 39
preface | [_forthcoming_]
press-release | [_forthcoming_]
referral | [_forthcoming_]
related-materials | [_forthcoming_]
section | [_forthcoming_]
sources | [_forthcoming_]
subsection | [_forthcoming_]
table-of-contents | [_forthcoming_]
volume-summary | [_forthcoming_]

### Deprecated
- chapter-introduction
- errata_document-numbering-error
- event
- maps
- undetermined

**Comment by WaxCylinderRevival** [comment]
As of January 10, 2021:

Current @subtype value | frequency
-- | --
about-frus-series | 39
acknowledgments | 2
additional-volumes | 39
appendix | 19
editorial-note | 8,495
editorial-policies | 4
errata | 54
foreword | 
graphic-materials | 0
historical-document | 301,681
index | 1,225
map-or-chart | 11
note-on-covert-actions | 44
preface | 396
press-release | 52
referral | 979
section | 15
sources | 251
subsection | 110
table-of-contents | 467
volume-summary | 25

### Deprecated
- chapter-introduction
- errata_document-numbering-error
- event
- historian-statement
- maps
- notes
- related-materials
- undetermined

**Comment by joewiz** [comment]
Closed by #300
